src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -268,7 +268,10 @@ export class ActivityFeedItem extends React.Component<Props> {
     const { amount, symbol, decimals }: AaveExtra = event.extra;
     if (!amount || !symbol) return '';
     const value = formatUnits(amount, decimals);
-    return getValueWithSymbol(formatAmount(value, getDecimalPlaces(symbol)), symbol, isPositive, !value);
+    return getFormattedValue(formatAmount(value, getDecimalPlaces(symbol)), symbol, {
+      isPositive,
+      doNotFormat: !value,
```

Comment:
```suggestion
      noSymbol: !value,
```

Also, is it possible to catch this with type checking?

Reply:
Uff ;/
Yeah. I'll add exact object type for this in a minute :)